core/main/src/service/apps/apps_updater.rs

```diff
@@ -92,28 +114,30 @@ pub async fn update(mut client: ExtnClient, apps_update: AppsUpdate, ignore_list
         }
     };
 
-    // Uninstall removed apps
+    if uninstalls_enabled {
```

Comment:
This is all whitespace changes, this just put all the uninstall code in a block that checks if it is enabled first.